test/test-ncu.js

```diff
@@ -25,14 +25,14 @@ describe('npm-check-updates', function () {
     describe('cli', function() {
 
         it('should accept stdin', function() {
-            return spawn('node', ['bin/npm-check-updates'], '{ "dependencies": { "express": "1" } }')
+            return spawn('node', ['bin/ncu'], '{ "dependencies": { "express": "1" } }')
                 .then(function (output) {
                     output.trim().should.startWith('express')
                 });
         });
 
         it('should output json with --jsonAll', function() {
-            return spawn('node', ['bin/npm-check-updates', '--jsonAll'], '{ "dependencies": { "express": "1" } }')
+            return spawn('node', ['bin/ncu', '--jsonAll'], '{ "dependencies": { "express": "1" } }')
```

Comment:
had to change; else, `JSON.parse` fails because of the extra `[INFO]` output that gets printed (since role of `bins` have [swapped](https://github.com/tjunnone/npm-check-updates/pull/203/files#r51649427)